package.json

```diff
@@ -23,6 +23,7 @@
     "clean": "rm -rf dist bundle.* index.html && mkdir dist",
     "build": "npm run clean && babel src -d dist --copy-files && node-sass src/main.scss main.css",
     "lint": "eslint src tests showcase --ignore-pattern node_modules --ignore-pattern bundle.js",
+    "lint-styles": "stylelint src/*.scss src/styles/*.scss --syntax scss",
```

Comment:
We should probably be fine with only `stylelint src/styles/*.scss --syntax scss`